### ISSUE TITLE
Nexus 6 Support

### ISSUE THREAD

**Issue by npjohnson** [efidroid/projectmanagement#82]
ARCH: APQ8084
OEM: Motorola
Branch used to build: LA.BR or LA.BF (both get the same result)

LK builds surprisingly quickly. Whether booted or flashed to the Boot partition, LK doesn't initialize, and it hangs at the Google (aboot) logo. No LK access over USB.

**Comment by npjohnson** [comment]
This relates to missing includes in the apq8084 target. Someone from Slack purportedly pull requested in the necessary changes.

**Comment by M1cha** [comment]
I've merged these changes. Still no idea about ur "compiles too fast" thing.

**Comment by npjohnson** [comment]
It isn't really important that is builds fast as it seems in the end. Seems like the fact that it boots on the Note 4 (also apq8084) proves that theory about why it wouldn't work wrong.

**Comment by npjohnson** [comment]
Now that we have merged apq8084 support, and it works (as per the Note 4 port's example), I tried building with these changes, and we still hang at the Google logo (weirdly with no unlocked padlock).

I tried adding both M1cha's recommended display define, and the 2NDSTAGE_DTB define, but neither change the outcome.

I also tried the new commits plus M1cha's manual FDT appendage method which proved to work for the 6P (obviously modified for this device to not use gzip compression as the Nexus 6 doesn't support gzip packed kernels), and we still hang at the Google logo with no LK access (again, weirdly with no unlocked padlock which appeared on the 6P when we got stuck at the Google logo).

I'll be providing weekly updates like this on the device's I am currently trying to port the project to from now on.
